axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/axon/EventBuffer.java

```diff
@@ -102,14 +104,12 @@ public EventBuffer(EventStream delegate,
                 lock.unlock();
             }
         });
+        logger.debug("Now, with adjusted poll operation!");
```

Comment:
I like the advertising ;-)

Reply:
Ha darn, I'll drop this as this wasn't intended to be part of the PR. Thanks for signalling it.